cvmfs/cvmfs.cc

```diff
@@ -50,6 +46,7 @@
 #include <sys/time.h>
 #include <sys/types.h>
 #include <sys/wait.h>
+#include <sys/xattr.h>  // NOLINT
```

Comment:
@christge-sft This should not be moved, although it could be reviewed if it's actually an issue.
Add  a `// clang-format off` block above